File: app/src/main/java/com/pds/blog/MainActivity.java
package com.pds.blog;

import android.os.Bundle;
import android.view.View;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;

import com.pds.application.FlutterHelper;

/**
 * @author: pengdaosong
 * CreateTime:  2019/3/16 4:47 PM
 * Email：pengdaosong@medlinker.com
 * Description:
 */
public class MainActivity extends AppCompatActivity {
    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    public void doFlutter(View view) {
        FlutterHelper.startFlutter(this);
    }
}

File: study/kotlin_demo/src/main/java/com/pds/demo/MainActivity.kt
package com.pds.demo

import android.app.Activity
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import com.pds.demo.expand.isPhoneNo

class MainActivity : Activity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        "1570846008".isPhoneNo()
    }
}